src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -1022,4 +1021,7 @@ export default {
 		background-color: transparent;
 	}
 }
+::v-deep .trigger {
+	display: flex;
```

Comment:
Does this work on the file sidebar and is obsoleting the margin i added there for the close sidebar handle?

Reply:
Not sure